src/state/members-candidates/reducers.js

```diff
@@ -31,7 +31,7 @@ const peopleReducer = (state = initialState, {
     case 'SET_SELECTED_MEMBER':
       return {
         ...state,
-        selectedMember: payload
+        selectedMember: payload.id,
```

Comment:
we already have loaded this data, we just need to store the id of the selected one. To make this better, could send the id as the payload instead of all the data.